.github/workflows/duplicate_videos.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash -x
+
+# This script is run from within the .github/workflows directory and so
+# all paths are relative.
+
+echo "Finding duplicate videos..."
+
+SRCDIR="../../dev/_static/images"
```

Comment:
I would consider using a command-line argument for the root directory, and then making all other searches relative to the root.